server/src/utils/healthcheck.ts

```diff
@@ -0,0 +1,21 @@
+const http = require("http");
+
+const options = {
+  host: "localhost",
```

Comment:
Would it be better to use the `/api/server-info/ping` endpoint?

Reply:
Yes, and we should probably also assert on the response body, not just the status code.

Reply:
### Expected value
I first tested with a wget call for a successful call to `/api/server-info/ping` gave. It responded with this:

```
2024-05-19 20:47:35 URL: http://localhost:3001/api/server-info/ping 200 OK
```
Next I attempted to read that data inside the healthcheck.ts file


### What I tried
I'm having some trouble using `node.js` `http.request()` function for the purpose of getting the response body. Looking at their [docs](https://nodejs.org/api/http.html#httprequesturl-options-callback). It seems something like this should work. When placed inside the callback function of `http.request()`:

```javascript
res.on('data', (chunk) => {
  console.log(`BODY: ${chunk}`);
});
```

but this never gets triggered. Not sure what I'm doing wrong here. If anyone is familiar with http.request(), I need some help on reading the body. :bow: 

### Alternative
I didn't see any http requesting packages. So, I would like to make http.request() work here.

This only other useful piece of data I saw in the response was `statusMessage`. Which has the value of "OK". This seems to be like it could be used as a replacement for the body. Any thoughts on this idea?

Reply:
Just use fetch instead

Reply:
oh nice fetch does work. Seems I have to do a weird `signal` thing to implement a timeout :face_exhaling: 

But I was able to read the body this way.